service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseManagementOrderGenerationService.java

```diff
@@ -51,6 +49,13 @@
     private final CaseDataExtractionService dataExtractionService;
     private final DraftCMOService cmoService;
 
+    @Autowired
```

Comment:
why not `@RequiredArgsConstructor(onConstructor = @__(@Autowired))` annotation on the class?

Reply:
not sure how this happened. I think i was trying delombok and forgot to revert. I will change it back.